doc/source/getting_started/notebooks/ODL Basics.ipynb

```diff
@@ -0,0 +1,662 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Vector space F^n with vector multiplication.\n",
+      "\n",
+      "    This space implements n-tuples of elements from a `Field` ``F``,\n",
+      "    which is usually the real or complex numbers.\n",
+      "\n",
+      "    Its elements are represented as instances of the `NumpyFnVector` class.\n",
+      "    \n"
+     ]
+    }
+   ],
+   "source": [
+    "print(s.__doc__)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This text actually gives quite a lot of information. It tells us that our object `s` created by the command `odl.fn(3, 'int')`\n",
+    "- is a [vector space](https://en.wikipedia.org/wiki/Vector_space),\n",
+    "- contains vectors of a fixed length `n` (that's the \"n\" in `Fn`),\n",
+    "- is constructed over the <a href=\"https://en.wikipedia.org/wiki/Field_(mathematics)\">field</a> (the \"F\" in `Fn`) of real or complex numbers (usually) and\n",
+    "- represents its elements as `NumpyFnVector` instances.\n",
+    "\n",
+    "The last part is easiest to check since we already have an element `x` to test:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "odl.space.npy_ntuples.NumpyFnVector"
+      ]
+     },
+     "execution_count": 16,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "type(x)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Indeed, the type is the same as the one mentioned in the docstring. We should be able to probe also the other properties."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "RealNumbers()"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "s.field"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 18,
+   "metadata": {
+    "collapsed": false
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "3"
+      ]
+     },
+     "execution_count": 18,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "s.size  # the equivalent of \"length\", NumPy users will recognize it"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Also the word \"contains\" resonates with experienced Python users. They think of the `__contains__` method and want to probe it with the `object in container` expression. Here, the vector should be in the space:"
```

Comment:
Way to much words, simply call `x in s` without the long introduction. I guess you can keep "Here, the vector should be in the space:"